### DISCUSSION TITLE
LDM Notes for June 17th, 2024

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#8226] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-06-17.md

### Agenda

- `params` Span breaks
- Overload resolution priority questions
    - Application error or warning on `override`s
    - Implicit interface implementation
- Inline arrays as `record struct`s

**Comment by KennethHoff** [comment]
In regards to overload resolution and interfaces. If I understood correctly, you can apply the Attribute to interface methods, but implementors will not use it. What does that mean in practice? 

Calling an implementors implicit implementation of such a method would disregard the Attribute, but calling it explicitly (by casting it, or via generic constraints, or similar) would be affected by the Attribute?

**Reply by 333fred** [reply]
Yes, that is an accurate summary of the example scenario that is in the open issue link 🙂.
